plugins/hls-fourmolu-plugin/src/Ide/Plugin/Fourmolu.hs

```diff
@@ -25,7 +24,7 @@ import           Ide.Types
 import           Language.LSP.Server         hiding (defaultConfig)
 import           Language.LSP.Types
 import           Language.LSP.Types.Lens
-import           "fourmolu" Ormolu
+import           Ormolu
```

Comment:
I once removed it in #1836, but I thought that this may confuse people...

Reply:
BTW, you can also remove it from Ormolu if you prefer

Reply:
Sure: 7bb5a9c

Reply:
> I once removed it in #1836, but I thought that this may confuse people...

My thinking is that with the new plugin architecture, those two plugins no longer know anything about each other, so it doesn't really make sense to keep the qualified imports.

In fact, has there been any discussion about the long-term plan for where the plugins live? I wouldn't mind if the Fourmolu plugin was moved to [the Fourmolu organisation](https://github.com/fourmolu), for example.

Reply:
AFAIK there has been no such discussion, and I'm afraid that would increase maintenance cost significantly... but it is definitely worthwhile to discuss that with other maintainers.